src/test/regress/expected/columnar_chunk_filtering.out

```diff
@@ -1066,3 +1066,32 @@ RESET columnar.max_custom_scan_paths;
 RESET columnar.qual_pushdown_correlation_threshold;
 RESET columnar.planner_debug_level;
 DROP TABLE pushdown_test;
+--
+-- https://github.com/citusdata/citus/issues/5803
+--
+
+create table clause_check (id int, company_name text) using columnar;
+INSERT INTO clause_check VALUES(1, 'AAPL');
+INSERT INTO clause_check VALUES(2, 'IBM');
+INSERT INTO clause_check VALUES(3, 'TSLA');
+INSERT INTO clause_check VALUES (generate_series(4,100000), md5(random()::text));
```

Comment:
It seems like this won't really be a good test because the md5s will be totally uncorrelated to the physical order. Or is the point that the md5s are always lower case?

Also, randomness could make the test non-deterministic.

Try to make tests something where we can look at the test and tell what the answer should be. In some cases, tests are complex and a comment can help.

That's not a rule for all tests; often it's good just to know when something changes (e.g. a plan) without necessarily saying one result is correct and the other is wrong.